spec/models/booking_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Booking, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :start_time }
+  end
```

Comment:
Shouldn't bookings belong to a facility? How will you know what you are booking for?